docs/integrations/spark/installation.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
```

Comment:
Shouldn't the installation be at the very top? I thought the rest relies on it heavily 😄